versions.json

```diff
@@ -1,4 +1,4 @@
 [
-  "1.2",
-  "1.1"
+  "1.2.0",
```

Comment:
If we specify the patch version number here, it will require us update the version number per every patch release. However, we probably not need to do this since 1.2.1 and 1.2.0 docs shouldn't differ according to SemVer because there are no new features in a new patch release. 

I would suggest keeping it as minor version or use `1.1.x`, `1.2.x`.

Reply:
Done